x/ccv/provider/keeper/keeper_test.go

```diff
@@ -832,32 +833,63 @@ func TestDenylist(t *testing.T) {
 	require.True(t, providerKeeper.IsDenylistEmpty(ctx, chainID))
 }
 
-func TestMinimumPowerInTopN(t *testing.T) {
+// Tests setting, getting and deleting number parameters that are stored per-consumer chain.
+// The tests cover the following parameters:
+// - MinimumPowerInTopN
+// - MinStake
+// - MaxValidatorRank
+func TestKeeperIntConsumerParams(t *testing.T) {
```

Comment:
This test seems inconsistent with the way we test other params (e.g., `TestValidatorSetCap`, `TestValidatorsPowerCap`, etc.) where we have one test per param.

Reply:
This is just a more concise way to test it imo, one table-based test with N cases vs N tests that repeat the same logic

Reply:
Actually, I'll move the other params that use integers here, too, unless you see a reason to prefer the separate tests?

Reply:
This was more of a nit comment. You can leave as is or change it to include other integer params.I just wanted to mention it.